src/chains/ethereum/ethereum/tests/api/eth/getBlockByNumber.test.ts

```diff
@@ -0,0 +1,35 @@
+import assert from "assert";
```

Comment:
Can you add a tests that change the starting default difficulty.

Also, lets test the increasing `totalDifficulty` (both with and without the default difficulty).

Reply:
sounds good to me